Mathlib/Analysis/Meromorphic/Divisor.lean

```diff
@@ -116,4 +126,26 @@ theorem divisor_inv [CompleteSpace 𝕜] {f : 𝕜 → 𝕜} :
   · simp [divisor_apply, h, (h.1 z h.2).order_inv]
   · simp [divisor_def, h]
 
+/-- Adding an analytic function to a meromorphic one does not change the pole divisor. -/
+theorem divisor_add_analytic {f₁ f₂ : 𝕜 → E} (hf₁ : MeromorphicOn f₁ U)
```

Comment:
```suggestion
theorem negPart_divisor_add_of_analyticNhdOn_right {f₁ f₂ : 𝕜 → E} (hf₁ : MeromorphicOn f₁ U)
```
and can we have the `left` version as well?

Reply:
Done.